src/javascript/app_2/Modules/Trading/Components/Elements/full_screen_dialog.jsx

```diff
@@ -70,10 +70,11 @@ const FullScreenDialog = (props) => {
 };
 
 FullScreenDialog.propTypes = {
-    children: PropTypes.any,
-    onClose : PropTypes.func,
-    title   : PropTypes.string,
-    visible : PropTypes.bool,
+    children        : PropTypes.any,
+    onClose         : PropTypes.func,
+    title           : PropTypes.string,
+    visible         : PropTypes.bool,
+    wrapperClassName: PropTypes.string,
```

Comment:
for string type I believe we separate by `_`

Reply:
I think we use camelCase when it's in props, should I pass wrapper_class in props?